RFS/src/testFixtures/java/com/rfs/framework/SearchClusterContainer.java

```diff
@@ -107,20 +83,34 @@ public String getUrl() {
 
     @Override
     public void close() {
-        log.info("Stopping ElasticsearchContainer version:" + version.prettyName);
+        log.info("Stopping container version:" + version.prettyName);
         log.debug("Instance logs:\n" + this.getLogs());
         this.stop();
     }
 
     public enum INITIALIZATION_FLAVOR {
```

Comment:
I'm not convinced that enums storing more than integral values is a good pattern.  In some cases, this seems natural - maybe this is one of them.  Please move this toward the top of the file since it has some static constant policies baked into it.